src/main/kotlin/com/spectralogic/rioclient/Broker.kt

```diff
@@ -122,6 +122,7 @@ data class ObjectResponse(
     val size: Long,
     val creationDate: String,
     val broker: String,
+    val checksum: String,
```

Comment:
Is it this simple?  From RioBroker the checksum comes back as more than just a single string:

    "checksum": {
        "hash": "",
        "type": "NONE"
    }

Reply:
You're right, fixing.